### DISCUSSION TITLE
Error installing

### DISCUSSION THREAD

**Discussion by chi28170** [mak-gitdev/HA_enoceanmqtt#147] [Q&A]
I am trying to install the addon and keep getting the error ive posted.  
![enoceanMqttError](https://github.com/mak-gitdev/HA_enoceanmqtt/assets/10568227/824af7e6-be98-4c00-8bcd-db08da8aac84)

**Comment by Kohhal** [comment]
Same error here, did you figure it out?:

The command '/bin/ash -o pipefail -c apk add --no-cache python3 py3-pip git && python3 -m venv /app/venv && . /app/venv/bin/activate && python3 -m pip install --upgrade pip && pip3 install pyyaml && pip3 install tinydb && pip3 install paho-mqtt==1.6.1 && pip3 install git+https://github.com/mak-gitdev/enocean.git && git clone -b master --single-branch --depth 1 https://github.com/embyt/enocean-mqtt.git && cd enocean-mqtt && python3 setup.py develop && cd .. && if [ "${BUILD_VERSION}" = "head" ]; then echo "Installing from repository head" && git clone -b develop --single-branch --depth 1 https://github.com/mak-gitdev/HA_enoceanmqtt.git ; else if [ -z "${BUILD_VERSION##*.*.*}" ]; then echo "Installing development version ${BUILD_VERSION%-*}" ; else echo "Installing stable version ${BUILD_VERSION%-*}" ; fi; wget -nv -O "/app.tar.gz" "https://github.com/mak-gitdev/HA_enoceanmqtt/archive/refs/tags/${BUILD_VERSION%-*}.tar.gz" && echo "Extracting /app.tar.gz" && tar xzf "/app.tar.gz" && rm "/app.tar.gz" && mv -v "HA_enoceanmqtt-${BUILD_VERSION%-*}" HA_enoceanmqtt ; fi; cp -rf HA_enoceanmqtt/enoceanmqtt enocean-mqtt && rm -rf HA_enoceanmqtt' returned a non-zero code: 1

**Comment by valumons** [comment]
j'ai le même problème .Quelqu'un a un solution?

**Reply by ChristopheHD** [reply]
Le log n'est pas assez précis pour savoir ce qui a cloché, mais j'ai souvent vu des personnes avoir des problèmes à cause du DNS, donc vérifie la configuration DNS de ton système et retente.
Autre info : ce dépôt est abandonné, tu trouveras des fork en cherchant.
